plugins/demoproxy.c

```diff
@@ -0,0 +1,218 @@
+/*
+ *      demoproxy.c - this file is part of Geany, a fast and lightweight IDE
+ *
+ *      Copyright 2007-2012 Enrico Tröger <enrico(dot)troeger(at)uvena(dot)de>
+ *      Copyright 2007-2012 Nick Treleaven <nick(dot)treleaven(at)btinternet(dot)com>
```

Comment:
Since you wrote this new file, it would be more appropriate to put your copyright in there.